tools/fileinfo/features/pe-loader-corruptions/test.py

```diff
@@ -9,6 +9,7 @@ class Test000(Test):
             '000-correct-file-coff-debug-info-32bit.ex_',
             '000-correct-file-small-alignment-32bit.ex_',
             '000-correct-file-size-of-opt-header-zero-64bit.ex_',
+            '000-valid-file-weird-virtual-size.ex_',
```

Comment:
Out of curiosity, is there a reason why the newly added file begins with `000-valid-` while all the other ones begin with `000-correct`? Maybe this is intended. I would just like to confirm.

Reply:
I am gonna add all other valid files discovered in the future as "000-valid*", so they are groupped together.